docs/user/ml/index.asciidoc

```diff
@@ -98,3 +98,44 @@ image::user/ml/images/outliers.png[{oldetection-cap} results in {kib}]
 
 For more information about the {dfanalytics} feature, see
 {ml-docs}/ml-dfanalytics.html[{ml-cap} {dfanalytics}].
+
+[[xpack-ml-aiops]]
+== AIOps
+
+AIOps is a part of {ml-app} in {kib} which provides features that use advanced 
+statistical methods to help you interpret your data and its behavior.
+
+[discrete]
+[[explain-log-rate-spikes]]
+=== Explain log rate spikes
+
+preview::[]
+
+Explain log rate spikes is a feature that uses advanced statistical methods to 
+identify reasons for increases in log rates. It makes it easy to find unusual 
+spikes by using the investigative workflow view. Examine the histogram chart of 
+the log rates for a given {data-source}, and find the reason behind a particular 
+change possibly in millions of log events across multiple fields and values.
```

Comment:
> It makes it easy to find unusual spikes by using the investigative workflow view. 

Though "find the reason" is mentioned in the sentence after that, but I wonder if just "find the spikes" is a bit too underselling because it's not just about the spike but the fields and values that influence it. Maybe something like:

> It makes it easy to find and investigate causes of unusual spikes by using the analysis workflow view.